pages/about.tsx

```diff
@@ -40,30 +40,31 @@ function About(props: AboutProps) {
 
                     <h4 className="alinea-title">The story</h4>
                     <p className="light alinea">
-                        Friends came to me enthusiastic about mechanical keyboards. I'm always in to help them find
-                        their best keyboard. The second step is ofcource, a nice keycapset!
+                        Excited friends came to me with questions about mechanical keyboards. I'm always in to help them
+                        find a keyboard that suits them best. The second step is a nice keycapset ofcourse!
```

Comment:
```suggestion
find a keyboard that suits them best. The second step is a ofcourse... a nice keycapset!
```

Reply:
Als je dit grammaticaal correct zou schrijven krijg je:
"The second step is, ofcourse, a nice keycapset!"